code/modules/language/language_holder.dm

```diff
@@ -378,6 +378,17 @@ Key procs
 							/datum/language/terrum = list(LANGUAGE_ATOM),
 							/datum/language/narsie = list(LANGUAGE_ATOM))
 
+/datum/language_holder/grod
```

Comment:
add to silicons

Reply:
I deliberately did not give the Grod tongue to Silicons when creating them.